build/tools/templates/Archive/Archive.csproj

```diff
@@ -1,4 +1,10 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project>
+  <PropertyGroup>
+    <ImportDirectoryBuildProps>false</ImportDirectoryBuildProps>
```

Comment:
Curious what this change is for?

Reply:
By moving Directory.Build.props up to the root of the project, it had a wide impact on the .csproj files we use. I just updated with some cleanup that I think makes it easier to keep our .csproj files minimal.